src/main/groovy/com/ullink/gradle/opencover/OpenCoverNUnitPlugin.groovy

```diff
@@ -27,6 +29,15 @@ class OpenCoverNUnitPlugin implements Plugin<Project> {
         task.conventionMapping.map 'parallelTargetExecArgs', {
             intermediateNunitResultsPath.mkdirs()
             nunit.getTestInputAsList(nunit.where?.value).collect({nunit.buildCommandArgs(it,  new File (intermediateNunitResultsPath,  UUID.randomUUID().toString() + ".xml"))})
+
+        }
+
+        task.doLast {
```

Comment:
maybe do that *only* if running in parallel ? (otherwise it'll probably fail or do some mess for non-parallel runs)

other than that, :shipit:

Reply:
fair enough, thanks !